### ISSUE TITLE
Организовать структуру проекта

### ISSUE THREAD

**Issue by buftio** [gnkoshelev/ai-game-platform#1]
Платформа подразумевает минимум две sdk
- для тех, кто будет делать игры
- для тех, кто будет делать AI

So Нужно разделить исходный код на библиотеки. Если приерно, то такие:
- application kernel
- web hooks (? можно для MVP не отделять от предыдущего)
- game dev. sdk
- ai dev. sdk

**Comment by buftio** [comment]
* Для примера раскидал то, что у нас было вот так:

  ![inapp](https://cloud.githubusercontent.com/assets/14301123/24247991/6df76b74-0fef-11e7-8523-e6285ac56459.png)

* И создал проект игры в котором наш пакет platform используется, но в его составе нет пакета app (в котором исходники самой платформы). В нём только игровые интерфейсы и глобально используемые структуры данных.

  ![gameusage](https://cloud.githubusercontent.com/assets/14301123/24247994/70fbb050-0fef-11e7-841f-c5602482c028.png)

**Comment by gnkoshelev** [comment]
Предложение дельное.

Если game sdk и ai sdk имеют общие части, то их тоже стоит вынести в общую (common) библиотеку.

**Comment by buftio** [comment]
Да, примерно так.  В частности, общие интерфейсы, структуры данных, математические(?) библиотеки будут там.
